### DISCUSSION TITLE
Change reading width

### DISCUSSION THREAD

**Discussion by rodneyljr** [ezra-bible-app/ezra-bible-app#1154] [Ideas]
I love this app. I use Debian so it's hard to find Bible software that kicks butt like this. I just have one question. It seems the reading mode width is very narrow. Is there a way to change this? Maybe I'm missing something. 
![Screenshot from 2024-11-16 20-05-18](https://github.com/user-attachments/assets/1d1232a3-d18c-40f3-9835-bb11e0252f43)

**Comment by tobias-klein** [comment]
Hi @rodneyljr,

Thank you for your feedback. 
The width of the text was actually limited in order to improve readability, see https://github.com/ezra-bible-app/ezra-bible-app/issues/815.

However, this is something which is depending on taste. We could maybe it configurable. How about that?

**Reply by tobias-klein** [reply]
I have just created an issue #1155 and implemented a new option with PR #1156. The option will then be available in the next release.

**Comment by rodneyljr** [comment]
Hey Tobias, I'm sorry for not responding to this. I'm new to Github and didn't even think to check it. Love that there was an option added to be configurable. Thanks for all of your work!
